### ISSUE TITLE
Non-WFS GeoJSON files don't error when imported as service

### ISSUE THREAD

**Issue by ShrutiVellanki** [fgpv-vpgf/fgpv-vpgf#3335]

**Comment by JahidAhmed** [comment]
Should we also throw an error if we attempt to import a WFS layer (example: http://geo.wxod-dev.cmc.ec.gc.ca/geomet/features/collections/hydrometric-stations/items?startindex=6000) as a file through the file URL option.

**Comment by ShrutiVellanki** [comment]
> Should we also throw an error if we attempt to import a WFS layer (example: http://geo.wxod-dev.cmc.ec.gc.ca/geomet/features/collections/hydrometric-stations/items?startindex=6000) as a file through the file URL option.

good point. i'll throw an error for all attempts to import a service as a file and files as a service.

**Comment by james-rae** [comment]
Will just comment that our desire to accurately predict what a pasted URL is pointing at has long been an outstanding issue, mainly as the ESRI request library doesn't support `HEAD` requests.
Our current approach is inefficient and unrobust.
See [this pair of TODO comments](https://github.com/fgpv-vpgf/geoApi/blob/v3.0.0-10/src/layer.js#L369)

**Comment by ShrutiVellanki** [comment]
> Should we also throw an error if we attempt to import a WFS layer (example: http://geo.wxod-dev.cmc.ec.gc.ca/geomet/features/collections/hydrometric-stations/items?startindex=6000) as a file through the file URL option.


> Will just comment that our desire to accurately predict what a pasted URL is pointing at has long been an outstanding issue, mainly as the ESRI request library doesn't support `HEAD` requests.
> Our current approach is inefficient and unrobust.
> See [this pair of TODO comments](https://github.com/fgpv-vpgf/geoApi/blob/v3.0.0-10/src/layer.js#L369)

will still leave this issue as outstanding and unassign myself because these points are probably worth addressing at a later date

**Comment by mweech** [comment]
User Added Layers requires a review to create updated specifications on how to handle input vs expected results.  It is recommended that we change our approach to attempt to import based on the users selection and catch failures as gracefully as possible rather than try to predict what the input type is or if it will be successful.

**Comment by AleksueiR** [comment]
Even if we remove wizard predictions, WCAG [3.3.1 Error Identification](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html) and [3.3.3 Error Suggestion](https://www.w3.org/WAI/WCAG21/Understanding/error-suggestion.html) guidelines still need to be addressed whenever possible.
